// ayla-npm/src/index.js
function qrcode(a) {
        if (a.message === null || a.message === undefined || a.message === '' || a.message === ' ') {
            console.log('Error: undefined message!')
        } else {
            let msg = a.message
            let qr = encodeURI(`https://api.qrserver.com/v1/create-qr-code/?data=${msg}`)
                return qr
      }
}

function mcbody(a) {
    if (a.message === null || a.message === undefined) {
        console.log('Error: undefined skin!')
    } else {
        let msg = a.message
        let mc = encodeURI(`https://mc-heads.net/body/${msg}}/500`)
            return mc
        }
}

function clyde(a) {
    if (a.message === null || a.message === undefined || a.message === '' || a.message === ' ') {
        console.log('Error: undefined message!')
    } else {
        let msg = a.message
        let msgclyde = encodeURI(`https://ctk-api.herokuapp.com/clyde/${msg}`)
            return msgclyde
        }
}

function google(a) {
    if (a.message === null || a.message === undefined || a.message === '' || a.message === ' ') {
        console.log('Error: undefined message!')
    } else {
        let msg = a.message
        let google = encodeURI(`https://www.google.com/search?q=${msg}`)
            return google
        }
}

function youtube(a) {
    if (a.message === null || a.message === undefined || a.message === '' || a.message === ' ') {
        console.log('Error: undefined message!')
    } else {
        let msg = a.message
        let yt = encodeURI(`https://m.youtube.com/results?search_query=${msg}`)
            return yt
        }
}


function captcha(a) {
    if (a.message === null || a.message === undefined || a.message === '' || a.message === ' ') {
        console.log('Error: undefined message!')
    } else {
        let msg = a.message
        let ct = encodeURI(`https://api.alexflipnote.dev/captcha?text=${msg}`)
            return ct
        }
}

function captcha2(a) {
    if (a.message === null || a.message === undefined || a.message === '' || a.message === ' ') {
        console.log('Error: undefined message!')
    } else {
        let msg = a.message
        let ct2 = encodeURI(`https://useless-api.vierofernando.repl.co/captcha?text=${msg}`)
            return ct2
        }
}

/*
function baby(a) {
    if (a.message === null || a.message === undefined || a.message === '' || a.message === ' ') {
        console.log('Error: undefined image!')
    } else {
        let img = a.image
        let bb = encodeURI(`https://useless-api.vierofernando.repl.co/baby?image='+${img}`)
            return bb
        }
}
*/
module.exports = {
    clyde,
    qrcode,
    google,
    youtube,
    captcha,
    captcha2,
    mcbody
}
